source/secure_tunneling.c

```diff
@@ -269,7 +267,7 @@ static int s_secure_tunneling_connect(struct aws_secure_tunnel *secure_tunnel) {
 }
 
 static int s_secure_tunneling_close(struct aws_secure_tunnel *secure_tunnel) {
-    if (secure_tunnel == NULL || secure_tunnel->stream_id == INVALID_STREAM_ID) {
```

Comment:
This function is to close the websocket. It doesn't matter if we currently have a valid stream.